docs/guides/fine_tuning.md

```diff
@@ -103,14 +103,18 @@ with open('customer_service_data.csv', 'w', newline='') as file:
 
 ## Making your data accessible to LLM Engine
 
-Currently, data needs to be uploaded to a publicly accessible web URL so that it can be read
-for fine-tuning. Publicly accessible HTTP and HTTPS URLs are currently supported.
-Support for privately sharing data with the LLM Engine API is coming shortly. For quick
-iteration, you can look into tools like Pastebin or GitHub Gists to quickly host your CSV
-files in a public manner. An example Github Gist can be found
-[here](https://gist.github.com/tigss/7cec73251a37de72756a3b15eace9965). To use the gist,
-you can use the URL given when you click the “Raw” button
-([URL](https://gist.githubusercontent.com/tigss/7cec73251a37de72756a3b15eace9965/raw/85d9742890e1e6b0c06468507292893b820c13c9/llm_sample_data.csv)).
+Currently, data needs to be uploaded to either a publicly accessible web URL or to LLM Engine's private file server so that it can be read for fine-tuning. Publicly accessible HTTP and HTTPS URLs are currently supported.
+
+To privately share data with the LLM Engine API, use LLM Engine's [File.upload](../../api/python_client/#llmengine.File.upload) API. You can upload data in local file to LLM Engine's private file server and then use the returned file ID to reference your data in the FineTune API.
+
+=== "Upload to LLM Engine's private file server"
+
+```python
+from llmengine import File
+
+response = File.upload(open("customer_service_data.csv", "r"))
+print(response.json())
```

Comment:
Perhaps we can add a comment that says it prints out a particular ID (e.g. you write `file-7DLVeLdN2Ty4M2m` below)? Then we can just reference that ID below. Well, IDs plural, if we do this for the `validation_file` as well.

Reply:
added note above